### ISSUE TITLE
Arrow annotation in legend

### ISSUE THREAD

**Issue by erythrocyte** [oxyplot/oxyplot#1697]
Is it possible to add arrow annotation to legend?
I mean something like in the picture below

![arrow_in_legend](https://user-images.githubusercontent.com/32854075/95945347-e9c23e00-0df2-11eb-826d-166f65424489.png)

**Comment by VisualMelon** [comment]
No, you would need to wrap them in a `Series` to be included in the Legend. I like at some point to add an `ILegendItem` abstraction so that we can use them more generally.

**Comment by erythrocyte** [comment]
> 
> 
> No, you would need to wrap them in a `Series` to be included in the Legend. I like at some point to add an `ILegendItem` abstraction so that we can use them more generally.

Could you please provide an example how can I wrap `Annotation` in a `Series`?
